src/deploy/functions/cache/hash.ts

```diff
@@ -8,7 +8,7 @@ import { getSecretVersions } from "../../../functions/secrets";
  * @param backend Backend of a set of functions
  */
 export function getEnvironmentVariablesHash(backend: Backend): string {
-  const hash = crypto.createHash("sha256");
+  const hash = crypto.createHash("sha1");
```

Comment:
nit* Does `sha1` HAVE to be consistent across all env vars? Just wondering if we should just define a helper function in this module to make sure we don't repeat `crypto.createHash("sha1");` again and again and instead just call `createHash()` helper function.

Reply:
Helper method it is!